src/Console/Command/BuildImage.php

```diff
@@ -210,6 +210,15 @@ protected function execute(InputInterface $input, OutputInterface $output): int
                     $dockerfilePath
                 )
             );
+        } elseif (self::FAILURE === $status) {
+            $io->writeln($process->getErrorOutput());
+            $io->error(
+                sprintf(
+                    'The %s tagged image had an error when building with the Dockerfile found in %s',
```

Comment:
A minor change please, because when error occured, the image is not yet tagged.

So the message should be:  'The %s image had an error when building with the Dockerfile found in %s'